src/hooks/index.js

```diff
@@ -43,8 +43,6 @@ var toInstrument = Object.create(null, {
       patches: {} } },
   'hapi': { enumerable: true, value: { file: './userspace/hook-hapi.js',
       patches: {} } },
-  'http': { enumerable: true, value: { file: './core/hook-http.js',
```

Comment:
Oh, I just realized what @AdriVanHoudt actually meant by this - @cristiancavalli, could you remove `hook-http.js`?